CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ SECURITY:
 BUG FIXES:
 
  * jobspec: Fixed validation of multi-region datacenters to allow empty region `datacenters` to default to job-level `datacenters`. [[GH-8426](https://github.com/hashicorp/nomad/issues/8426)]
+ * scheduler: Fixed a bug in Nomad Enterprise where canaries were not being created during multi-region deployments. [[GH-8456](https://github.com/hashicorp/nomad/pulls/8456)]
```

Comment:
```suggestion
 * scheduler: Fixed a bug in Nomad Enterprise where canaries were not being created during multi-region deployments. [[GH-8456](https://github.com/hashicorp/nomad/issues/8456)]
```

Reply:
Oops. There's no issue for this in OSS so I did mean to link it to the PR. But that should be _pull_ and not _pulls_. Fixed.